packages/closer/pages/roles/index.tsx

```
@@ -95,7 +101,9 @@ const RolesPage = ({ generalConfig, roles, error }: Props) => {
                                 {role.title}
                               </h3>
                               <p className="text-gray-600 mb-4">
-                                <span dangerouslySetInnerHTML={{ __html: role.description || '' }} />
+                                {initialRenderComplete && (
+                                  <span dangerouslySetInnerHTML={{ __html: role.description || '' }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)